### PR TITLE
Design the commands

### PR DESCRIPTION
**Pull request by suriyaa**
@nickrivadeneira @gkeary:

Hi guys, :smile:

I **designed the commands** in **`README.md`**. :pencil2:

_Yours,_
[**Suriyaa Kudo**](https://github.com/SuriyaaKudoIsc) :octocat:

### CODE REVIEW

**Comment by segiddins** [comment]
:+1:

**Comment by nickrivadeneira** [comment]
@SuriyaaKudoIsc LGTM, but I think you might be looking for the other Nick - @qrush.

File: README.md [hunk 1]
```diff
@@ -43,7 +45,9 @@ OR
 
 Include **'therubyracer'** in your Gemfile.
 
-    gem "therubyracer", :require => 'v8'
+```bash
```
Comment (nickrivadeneira) [inline README.md]: Is `bash` the right syntax highlighting choice here? Might need `ruby` or something.

Comment (suriyaa) [inline README.md]: @nickrivadeneira  These commands is default used by bash programs like CMD, terminal or command line.

Comment (nickrivadeneira) [inline README.md]: @SuriyaaKudoIsc That applies to the above commands (`bundle` and `rails server`), but I don't believe that applies to `gem "therubyracer", :require => 'v8'`. That line belongs in the `Gemfile`, which is Ruby.

The CLI command to install gems can take commands like `gem install therubyracer`, but that's not what's being displayed here in the docs.

Comment (suriyaa) [inline README.md]: @nickrivadeneira Oh! You're right!

**Comment by suriyaa** [comment]
@ffmike @gkeary @qrush @luislavena @dwradcliffe

**Comment by suriyaa** [comment]
@rubygems / @andremedeiros @arthurnn @bai @bronzdoc @drbrain @duckinator @dwradcliffe @evanphx @farukaydin @hsbt @indirect @krainboltgreene @ktheory @luislavena @LynnCo @qrush @segiddins @sferik @skottler @sonalkr132 @zzak

---

I'm still waiting for an answer.

**Comment by dwradcliffe** [comment]
This is also a deprecated project. No need to make any changes here.

**Pull request closed without merging**